[2024-10-30T17:59:21.069+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-30T17:59:21.074+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.create_table_user_review manual__2024-10-30T17:59:17.779423+00:00 [queued]>
[2024-10-30T17:59:21.076+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.create_table_user_review manual__2024-10-30T17:59:17.779423+00:00 [queued]>
[2024-10-30T17:59:21.076+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-30T17:59:21.080+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): create_table_user_review> on 2024-10-30 17:59:17.779423+00:00
[2024-10-30T17:59:21.082+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=70) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-30T17:59:21.083+0000] {standard_task_runner.py:72} INFO - Started process 73 to run task
[2024-10-30T17:59:21.083+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'create_table_user_review', 'manual__2024-10-30T17:59:17.779423+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/***.py', '--cfg-path', '/tmp/tmppf9o2k4f']
[2024-10-30T17:59:21.084+0000] {standard_task_runner.py:105} INFO - Job 49: Subtask create_table_user_review
[2024-10-30T17:59:21.097+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.create_table_user_review manual__2024-10-30T17:59:17.779423+00:00 [running]> on host 6bca134bba56
[2024-10-30T17:59:21.121+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='your_name' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='create_table_user_review' AIRFLOW_CTX_EXECUTION_DATE='2024-10-30T17:59:17.779423+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-30T17:59:17.779423+00:00'
[2024-10-30T17:59:21.122+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-30T17:59:22.081+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-30T17:59:22.086+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-30T17:59:22.087+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=create_table_user_review, run_id=manual__2024-10-30T17:59:17.779423+00:00, execution_date=20241030T175917, start_date=20241030T175921, end_date=20241030T175922
[2024-10-30T17:59:22.106+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-30T17:59:22.117+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-30T17:59:22.117+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
